src/authorizationProvider.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from "react";
+import authService from "./services/authService";
+
+const refreshToken = 'RefreshToken';
+const accessToken = 'AccessToken';
+const AuthorizationContext = React.createContext();
+class AuthorizationProvider extends Component {
+    state = {
+        loggedIn: null,
+        userFirstName: null,
+        userLastName: null,
+        userRole: null
+    }
+
+    async componentDidMount() {
+        if (localStorage.getItem(accessToken) !== 'null' &&
```

Comment:
ako sa to sprava, ked sa user prihlasi a nasledne da refresh?

Reply:
teto metoda componentDidMount je tu kvoli tomu pripadu.. jedine co ostane su v local storage tokeny.. pomocou nich sa urobi refresh ak su este validne vratia sa nove tokeny spolu s nejakymi zakladnymi udajami (meno, id). a teda mal by ostat prihlaseny uzivatel ktoremu ostali tokeny v local storage